### PR TITLE
Hungarian language update #09

### PR DESCRIPTION
**Pull request by big4billy**
Systemathic fuzzy removals everywhere

### CODE REVIEW

File: files/lang/hu.po [hunk 1]
```diff
@@ -5906,26 +5877,25 @@ msgstr ""
 "Ha iszol az oázisban, akkor felfrissülsz, de más nem történik. Csak az első "
 "csatádban segít."
 
-#, fuzzy
 msgid ""
 "A drink at the oasis fills your troops with strength and lifts their "
 "spirits. You can travel a bit further today."
 msgstr ""
-"Ha iszol az oázisban, akkor az egységeid ereje és hangulata feltöltődik. "
+"Ha iszol az oázisban, akkor az egységeid ereje és morálja feltöltődik."
```
Comment (ihhub) [inline files/lang/hu.po:5884]: Why a space was removed? It is a separation between 2 sentences.

File: files/lang/hu.po [hunk 2]
```diff
@@ -5906,26 +5877,25 @@ msgstr ""
...
 msgstr ""
-"Ha iszol a vizesgödörnél, akkor felfrissülsz, de más nem történik. Csak az "
+"Ha iszol az itatónál, akkor felfrissülsz, de más nem történik. Csak az "
 "első csatádban segít."
 
 msgid ""
 "A drink at the watering hole fills your troops with strength and lifts their "
 "spirits. You can travel a bit further today."
 msgstr ""
-"Ha iszol a vizesgödörnél, akkor az egységeid ereje és hangulata feltöltődik. "
```
Comment (ihhub) [inline files/lang/hu.po:5928]: Why a space was removed? It is a separation between 2 sentences.

File: files/lang/hu.po [hunk 3]
```diff
@@ -6980,267 +6948,204 @@ msgstr "Nekromancia"
...
 
-#, fuzzy
 msgid ""
 "%{skill} allows you to negotiate with monsters who are weaker than your "
 "group. "
 msgstr ""
-"A Fejlett Diplomácia lehetővé teszi a seregednél gyengébb lényekkel való "
-"tárgyalást. Mindannyian csatlakozhatnak hozzád."
+"A(z) %{skill} lehetővé teszi a seregednél gyengébb lényekkel való "
+"tárgyalást."
```
Comment (ihhub) [inline files/lang/hu.po:7106]: Please add a space at the end as there will be another sentence after.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @big4billy , I put several comments here. Could you please check them?

File: files/lang/hu.po (hunk 1 above)
Comment (big4billy) [reply files/lang/hu.po]: space added back

File: files/lang/hu.po (hunk 2 above)
Comment (big4billy) [reply files/lang/hu.po:5928]: space added back

File: files/lang/hu.po (hunk 3 above)
Comment (big4billy) [reply files/lang/hu.po]: one space added at the end.

**Review by ihhub — APPROVED** [review]

**Pull request merged**